src/execute_linter.cc

```diff
@@ -125,36 +104,52 @@ LinterResult ParseNoLintComments(absl::string_view sql,
   return result;
 }
 
-LinterOptions GetOptionsFromConfig() {
-  // TODO(orhanuysal): Connect here with a config file.
-  return LinterOptions();
-}
+LinterOptions GetOptionsFromConfig(Config config, absl::string_view filename) {
+  LinterOptions option(filename);
+  if (config.has_tab_size()) option.SetTabSize(config.tab_size());
+
+  if (config.has_end_line()) option.SetLineDelimeter(config.end_line()[0]);
+
+  if (config.has_line_limit()) option.SetLineLimit(config.line_limit());
+
+  if (config.has_single_quote()) option.SetSingleQuote(config.single_quote());
+
+  if (config.has_allowed_indent())
+    option.SetAllowedIndent(config.allowed_indent()[0]);
```

Comment:
Did you consider to have config field in LinterOptions class so here you can easily just set the field?